src/usecase/tui/app.rs

```diff
@@ -69,21 +69,28 @@ impl Model<'_> {
             AppState::SelectCommand(s) => match key.code {
                 KeyCode::Tab => Some(Message::MoveToNextPane),
                 KeyCode::Esc => Some(Message::Quit),
-                _ => match s.current_pane {
-                    CurrentPane::Main => match key.code {
-                        KeyCode::Down => Some(Message::NextCommand),
-                        KeyCode::Up => Some(Message::PreviousCommand),
-                        KeyCode::Enter => Some(Message::ExecuteCommand),
-                        _ => Some(Message::SearchTextAreaKeyInput(key)),
-                    },
-                    CurrentPane::History => match key.code {
-                        KeyCode::Char('q') => Some(Message::Quit),
-                        KeyCode::Down => Some(Message::NextHistory),
-                        KeyCode::Up => Some(Message::PreviousHistory),
-                        KeyCode::Enter | KeyCode::Char(' ') => Some(Message::ExecuteCommand),
-                        _ => None,
-                    },
-                },
+                _ => {
+                  let is_ctrl_pressed = key.modifiers == KeyModifiers::CONTROL;
```

Comment:
@ShoheiKamiya 
[nits]
How about modify this line to this to clarify the intension?

```suggestion
                  let is_ctrl_pressed = key.modifiers.contains(crossterm::event::KeyModifiers::CONTROL);

```

Reply:
@kyu08 I agree it! thanks!